README.md

```diff
@@ -1,68 +1,12 @@
 <!-- badges: start -->
   [![R build status](https://github.com/kelshmo/sageseqr/workflows/R-CMD-check/badge.svg)](https://github.com/kelshmo/sageseqr/actions)
 <!-- badges: end -->
-# sage-seqr
-RNASeq normalization pipeline with R.
+# RNA-seq normalization workflow in R
 
-This repository provides R markdowns for performing covariate adjustments and differential expression analysis of the RNA-seq data from AMP-AD and a Docker environment, thus all package dependencies are loaded. 
+The `sageseqr` package integrates the [`drake` R package](https://github.com/ropensci/drake/), the [`config` package for R](https://cran.r-project.org/web/packages/config/vignettes/introduction.html) and [Synapse](https://www.synapse.org/). `drake` tracks dependency relationships in the workflow and only updates data when it has changed. A `config` file allows inputs and parameters to be explicitly defined in one location. Synapse is a data repository that allows sensitive data to be [stored and shared responsibly](https://docs.synapse.org/articles/article_index.html#governance). 
 
-The results is an environment ready for analysis. 
-
-# Install Docker 
-
-1. [Find your supported platform and download Docker.](https://docs.docker.com/v17.12/install/#supported-platforms)
-
-# Run Container
-
-This Docker container is built on [`rocker/tidyverse`](https://hub.docker.com/r/rocker/tidyverse/) producing a debian stable work environment. 
-
-1. To get container for the first time: `docker pull kelshmo/covarr`
-2. Suggested configuration for running container: 
-```
-docker run -d -p 8787:8787 -e PASSWORD=<insert password> kelshmo/covarr #replace <password> with your password
-```
-- `-d` flag allows you the retain use of the terminal while the Docker image is running 
-- `-p` specifies port of choice
-- `-e` designates a password to login to RStudio
-
-If you are having trouble with your Docker, this [documentation](https://ropenscilabs.github.io/r-docker-tutorial/02-Launching-Docker.html) (or [this documentation](https://github.com/rocker-org/rocker/wiki/Using-the-RStudio-image)) may be useful.
-
-3. Now, RStudio has launched invisibly. Open a browser and enter your ip address followed by :8787. You should be greeted by the RStudio welcome screen.
-
-username: rstudio
-password: <password>
-  
-4. You can `docker stop <container name>` to pause your work and return to it later with `docker start <container name>`.
-
-* To find your container name `docker ps -a`
-
-# Clone the Repo
-
-`git clone https://github.com/kelshmo/sageseqr.git` to get the `sageseqr` tool in your Docker repository.
+The workflow takes RNA-seq gene counts and sample metadata as inputs, normalizes counts by CQN, removes outliers based on a user-defined threshold, empirically selects meaningful covariates and returns differential expression analysis results. 
```

Comment:
Maybe add citation for CQN

Reply:
Good point - I'm not sure I have seen software cited in a README. For now I will add a hyperlink, but look into example projects.